_topic_maps/_topic_map.yml

```diff
@@ -2294,8 +2294,19 @@ Topics:
     File: cpmso-getting-started
   - Name: Control plane machine set configuration
     File: cpmso-configuration
-  - Name: Using control plane machine sets
-    File: cpmso-using
+  - Name: Managing control plane machines with control plane machine sets
+    File: cpmso-managing-machines
+  - Name: Configuration options for control plane machines
```

Comment:
Are we planning to add VSphere too?

Reply:
Discussed in Slack, will rearrange some more for this :nerd_face: